### ISSUE TITLE
bp-scroll directive rendering scrollbars when there is no content to scroll

### ISSUE THREAD

**Issue by Goltergaul** [excellenteasy/bradypodion#97]
I have a div with the `bp-scroll` directive. As long as there is enough content in it so that you actually have to scroll, no scrollbars are shown. This is fine. But when there is not enough content, vertical and horizontal scrollbars show up. This applies to the android and the ios skin.

![screenshot_2014-07-22-10-08-33](https://cloud.githubusercontent.com/assets/34413/3655303/a67f2dac-1177-11e4-9f85-ec04b365f348.png)

**Comment by davidpfahler** [comment]
@Goltergaul First of all, thank you so much for submitting issues and helping us make bp better. 
Please try `bp-iscroll` directive and tell me if that works for you. bp-scroll is known to be a little buggy, so if that fixes it for you, I'll dig deeper into the issue.

**Comment by boennemann** [comment]
Hello @Goltergaul,

`bp-iscroll` is a directive as well, it is just distributed in a separate module to keep standard installations small.

More [here](http://docs.bradypodion.io/0.5.2/#/api/bp.iscroll).

You can use the service/provider to pass custom configuration to the iScroll script. http://docs.bradypodion.io/0.5.2/#/api/bp.iscroll.bpIscrollProvider

**Comment by Goltergaul** [comment]
@boennemann yeah figured it out after investigating the demo application ;) Maybe you should include that info in the documentation. But the good news is that it is working as it should :+1:
